packages/docs/src/helpers/navDataFormatUtils.ts

```diff
@@ -0,0 +1,132 @@
+import sortBy from 'lodash.sortby';
```

Comment:
Moved all the data restructuring to this file with some updated typing and docs.